uk.co.compucorp.civicrm.hrleaveandabsences/hrleaveandabsences.php

```diff
@@ -714,3 +714,25 @@ function _hrleaveandabsences_set_has_leave_approved_by_as_default_relationship_t
     );
   }
 }
+
+/**
+ * Implementation of the hook_civicrm_validateForm.
+ *
+ * @param string $formName
+ * @param array $fields
+ * @param array $files
+ * @param object $form
+ * @param array $errors
+ */
+function hrleaveandabsences_civicrm_validateForm($formName, &$fields, &$files, &$form, &$errors) {
```

Comment:
Remember this file has 2 sections: one to add hooks implementation and another one for helpers method. This is a hook, but it's in the helpers section. Please move it to the appropriate location